packages/typescript-estree/src/parser.ts

```diff
@@ -366,7 +366,7 @@ function parseAndGenerateServices<T extends TSESTreeOptions = TSESTreeOptions>(
   )!;
 
   /**
-   * Determine whether or not two-way maps of converted AST nodes should be preserved
+   * Determine whatever or not two-way maps of converted AST nodes should be preserved
```

Comment:
wasn't this correct?
Sounds strange now :thinking:

Reply:
hmm, you might be right, "whether or not" seems weird anyway 
> The addition of the "or not" is neither logically nor grammatically required. I think it's often used conversationally for emphasis. I definitely wouldn't use it in writing myself.
> https://english.stackexchange.com/questions/3382/whether-or-not-vs-whether

i think it will be safer to change it to
```suggestion
   * Determine if two-way maps of converted AST nodes should be preserved
```

> —used as a function word usually with correlative or or with or whether to indicate (1) until the early 19th century a direct question involving alternatives; (2) an indirect question involving stated or implied alternatives
https://www.merriam-webster.com/dictionary/whether

Reply:
just the "if" sounds good :+1: 
Or just "whether" (not "whatever" ;)